### PR TITLE
add "Google Cloud Loader 0.2.0 released" blog post

### PR DESCRIPTION
**Pull request by peel**
A short one to be in line with minor-version bump policy.

### CODE REVIEW

File: _posts/2019-12-11-google-cloud-loader-0.2.0-released/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+title: "Google Cloud Loader 0.2.0 released"
+title-short: Google Cloud Loader 0.2.0
+tags: [snowplow, gcs, gcp, loader ]
+author: Ben
+category: Releases
+permalink: /blog/2019/12/11/gcs-loader-0.2.0-released/
+---
+
+We are pleased to announce [version 0.2.0][release-020] of Google Cloud Storage loader.
+This small release is centered around critical bug fixes and updated default configuration.
```
Comment (goodits) [inline _posts/2019-12-11-google-cloud-loader-0.2.0-released/index.md]: This is missing a "that allows you to....". If I'm recalling correctly, this is the release that brings the GCS loader performance up to standard to sink events from the collector, right? Would be good to pull that out as a benefit, in addition to any performance stats we have still.

File: _posts/2019-12-11-google-cloud-loader-0.2.0-released/index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+<h2 id="roadmap">2. Roadmap</h2>
+
+GCS loader continues to evolve at Snowplow. In the next release are going to introduce improved data partitioning based upon data types.
+
+If you have other features in mind, feel free to log an issue in [the GitHub repository][gcs-loader-issues].
+
+<h2 id="contributing">3. Contributing</h2>
+
+You can check out the [repository][gcs-loader-repo] if you'd like to get involved!
```
Comment (goodits) [inline _posts/2019-12-11-google-cloud-loader-0.2.0-released/index.md]: Are we missing the Getting Help section? Should be a copy/paste job mostly.

File: _posts/2019-12-11-google-cloud-loader-0.2.0-released/index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+<h2 id="shards">1. Default shard number</h2>
+
+Google Dataflow is Google's job runner implementation that allows for "zero-configuration" scaling.
+The autoscaling enables dynamic worker number adjustment to the needs of a pipeline.
+Dataflow automatically detects imbalances in work assignments and can dynamically reassign underused workers to decrease job's overall processing time. However, setting a fixed number of shards (maintained throughout the job run) can negatively impact the dynamic work reassignment by either scattering data too much or throttling writes. Therefore making runner control the number of shards makes a better default and should comply to any environment that it is being run.
+
+To preserve the old behaviour set `--numShards=1` when running the job.
+
+<h2 id="roadmap">2. Roadmap</h2>
```
Comment (goodits) [inline _posts/2019-12-11-google-cloud-loader-0.2.0-released/index.md]: Do we need an upgrading section? Per https://snowplowanalytics.com/blog/2019/10/11/iglu-server-0.6.0-released/

File: _posts/2019-12-11-google-cloud-loader-0.2.0-released/index.md (hunk 1 above)
Comment (peel) [reply _posts/2019-12-11-google-cloud-loader-0.2.0-released/index.md]: I added more info on performance. The bottom line is that performance should no longer be outrageously low but the application itself has a strong (performance) [competitor](https://cloud.google.com/blog/products/data-analytics/simple-backup-and-replay-streaming-of-events-using-cloud-pubsub-cloud-storage-and-cloud-dataflow) that we won't be able to beat when it comes to raw numbers. So I want the impression to be that the performance is way better (not ~10% of collector throughput) and definitely worth pointing out but it's not something we can brag about.

Comment (goodits) [reply _posts/2019-12-11-google-cloud-loader-0.2.0-released/index.md]: Makes sense. I think what you've added speaking to the reason why we've made the change is more than enough. Thanks.

**Pull request closed without merging**